src/hooks/useENSRegistrationForm.ts

```diff
@@ -267,7 +253,7 @@ export default function useENSRegistrationForm({
     disabled,
     errors,
     isEmpty: empty,
-    isLoading,
+    isLoading: profileQuery.isLoading && isEmpty(values),
```

Comment:
Is there a reason this was changed?

I'm getting this now:

https://user-images.githubusercontent.com/7336481/165413174-ae8cd993-8bcd-4f05-9897-aab92696a720.mp4

Reply:
Note: The above shows a brief empty state & the action bottom sheet takes a while to instantiate.

Reply:
hm i'll roll it back, saw @benisgold fixed the jumping loading state issue already i was trying to fix that